### DISCUSSION TITLE
what to do with legacy

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [taiga-family/taiga-ui#9292] [General]
I want to use something like 

`
<tui-combo-box tuiDropdownMobile [tuiTextfieldCleaner]="true" formControlName="...">
                Country
                <tui-data-list-wrapper *tuiDataList [items]="countries | tuiFilterByInput" />
            </tui-combo-box>
`

is there something similar in non-legacy package or should i use legacy?

**Comment by waterplea** [comment]
For now you should keep using legacy package, they are not deprecated YET, just moved to free up space for us to re-implement them with new approach. We are waiting for the design team to finish the specs in case something has to change. When you see them actually deprecated with a strikethrough line — that means a new version was finalized and is ready to use.

**Comment by hakimio** [comment]
The closest you can get with `Textfield` from `core` package: https://stackblitz.com/edit/angular-6sncgm?file=src%2Fapp%2Fapp.template.html

**Reply by vytautas-pranskunas-** [reply]
> The closest you can get with `Textfield` from `core` package: https://stackblitz.com/edit/angular-6sncgm?file=src%2Fapp%2Fapp.template.html

the only problem with this that Chevron is :after and it does not get cursor:pointer...

**Reply by hakimio** [reply]
You can set cursor pointer on the input if you want:
```css
tui-textfield input {
  cursor: pointer;
}
```

**Reply by vytautas-pranskunas-** [reply]
> You can set cursor pointer on the input if you want:
> 
> ```css
> tui-textfield input {
>   cursor: pointer;
> }
> ```

this would lead whole input with curso - but it is typing input so pointer would be great only on chevron :) nevermind

**Reply by waterplea** [reply]
Our common icons mechanism is `tuiIcons` directive which uses `::before/::after` elements to display icons. So you can do this:
```css
tui-textfield::after {
  pointer-events: auto;
  cursor: pointer;
}
```
Don't forget to move `(click)` handler to `tui-textfield` otherwise it won't work.

**Reply by vytautas-pranskunas-** [reply]
This pointer-events: auto; not a good idea because it prevents dropdown to open

**Reply by hakimio** [reply]
If you put the click handler on `tui-textfield` it opens just fine.

**Reply by vytautas-pranskunas-** [reply]
thx
